README.md

```diff
@@ -385,6 +385,20 @@ If you want to help out, see CONTRIBUTING for a set of guidelines.
 
 This section contains general instructions for debugging failed installs or problems encountered with Monero. First ensure you are running the latest version built from the github repo.
 
+## Obtaining Stack Traces in Linux
+
+We generally use the tool `gdb` to provide stack trace functionality, althout this is currently being integrated into the easylogger++ system by @moneromooo-monero to provide stack traces across all systems
```

Comment:
gdb is for post mortem stack traces, or instantaneous ones.
easylogging++ is for programming logging of stack traces, including on exceptions. It won't replace using gdb.